### PR TITLE
Vidyo: Post-process event metadata

### PR DESCRIPTION
**Pull request by pferreir**
Related to https://github.com/indico/indico/pull/3840

### CODE REVIEW

**Comment by pferreir** [comment]
Closes #68

File: vc_vidyo/indico_vc_vidyo/plugin.py [hunk 1]
```diff
@@ -123,6 +124,14 @@ def icon_url(self):
     def _extend_indico_cli(self, sender, **kwargs):
         return cli
 
+    def _event_metadata_postprocess(self, sender, event, data, **kwargs):
+        associations = event.vc_room_associations
+        if associations:
+            description = event.description
+            urls = '\n'.join([assoc.vc_room.vidyo_extension.get_join_url(use_chooser=False) for assoc in associations])
```
Comment (ThiefMaster) [inline vc_vidyo/indico_vc_vidyo/plugin.py]: no need for the `[]`

File: vc_vidyo/indico_vc_vidyo/plugin.py [hunk 2]
```diff
@@ -123,6 +124,14 @@ def icon_url(self):
...
+            return {'description': '\n\n'.join(filter(None, [description, urls]))}
```
Comment (ThiefMaster) [inline vc_vidyo/indico_vc_vidyo/plugin.py]: `filter(None, ...)` is gone in Python 3 - maybe better to not start using such things now?

**Comment by pferreir** [comment]
Keeping it on hold until I speak to @gasparuben

**Pull request closed without merging**